### ISSUE TITLE
gradient_checkpoint=True

### ISSUE THREAD

**Issue by Cathrii** [Liuziyu77/Visual-RFT#198]
当开启地图检查点时，无论使用lora或全量微调，都会出现警告：UserWarning: None of the inputs have requires_grad=True. Gradients will be None。请问这是否会影响模型的训练？
